data/data/powervs/cluster/power_network/variables.tf

```diff
@@ -17,3 +17,15 @@ variable "vpc_crn" {
   type        = string
   description = "The CRN of the IBM Cloud VPC."
 }
+
+variable "pvs_network_name" {
+  type        = string
+  description = "The name of a pre-created Power VS DHCP Network."
+  default     = ""
```

Comment:
lets remove the default fields if there is nothing to set.

Reply:
maybe this is a terraform knoweldge gap for me, but if these arent set to an empty string, they're null, and i thought i'd get a type mismatch later when trying to check if it equals `nil` or not. i do have to check those to set some resource `count`s later on